.eslintrc.js

```diff
@@ -1,4 +1,4 @@
-const dictionary = require('./.eslint-dictionary.json');
+const dictionary = require("./.eslint-dictionary.json");
```

Comment:
we have a rule to prefer `'` over '"' . should this apply to this file as well ?